com.gitlab.j0chn.nextcloud_password_client.json

```diff
@@ -0,0 +1,90 @@
+{
+    "app-id": "com.gitlab.j0chn.nextcloud_password_client",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "41",
+    "sdk": "org.gnome.Sdk",
+    "command": "nextcloud_password_client",
+    "separate-locales": false,
+    "finish-args": [
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--socket=pulseaudio",
+        "--share=network",
+        "--device=all",
+        "--filesystem=xdg-download"
+    ],
+    "modules": [
+        {
+            "name": "libjsoncpp",
+            "buildsystem": "cmake",
+            "config-opts": [
+                "-DCMAKE_BUILD_TYPE=release",
+                "-DBUILD_SHARED_LIBS=ON"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://github.com/open-source-parsers/jsoncpp/archive/refs/tags/1.7.5.tar.gz",
+                    "sha256": "4338c6cab8af8dee6cdfd54e6218bd0533785f552c6162bb083f8dd28bf8fbbe"
+                }
+            ]
+        },
+
+        {
+            "name": "libsodium",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://download.libsodium.org/libsodium/releases/libsodium-1.0.18.tar.gz",
+                    "sha256": "6f504490b342a4f8a4c4a02fc9b866cbef8622d5df4e5452b46be121e46636c1"
+                }
+            ]
+        },
+        {
+            "name": "libsecret",
+            "buildsystem": "meson",
+            "config-opts": [
+                "-DCMAKE_BUILD_TYPE=release",
+                "-DBUILD_SHARED_LIBS=ON"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://github.com/GNOME/libsecret/archive/refs/tags/0.20.4.tar.gz",
+                    "sha256": "ca34e69b210df221ae5da6692c2cb15ef169bb4daf42e204442f24fdb0520d4b"
+                }
+            ]
+        },
+        {
+            "name": "nextcloud_password_client",
+            "buildsystem": "simple",
+            "only-arches": [
+                "x86_64"
+            ],
+            "build-commands": [
```

Comment:
given the layout of the tarball, I have doubt this will work.

Reply:
I am way too unexperienced to tell that.
I copied most of my configurations from fluffychat.
But I think that we will see that ;-)